schema/v1.0/ast/common.go

```diff
@@ -22,4 +23,4 @@ type RenameAttributes struct {
 // AttributeMap corresponds to a section representing a mapping of attribute names.
 // The keys are the old attribute name used in the previous version, the values are the
 // new attribute name starting from this version.
-type AttributeMap map[string]string
+type AttributeMap map[types.AttributeName]types.AttributeName
```

Comment:
This is a change to a public API, we should be careful and only do this if absolutely needed.

Reply:
Reverted.